src/lib/viewers/media/MediaControls.js

```diff
@@ -67,6 +68,8 @@ const FILMSTRIP_FRAME_HEIGHT = 90;
         this.setupSettings();
         this.setupScrubbers();
         this.attachEventHandlers();
+
+        this.hasTouch = Browser.hasTouch();
```

Comment:
Forgot it was defined in base, I removed it elsewhere. It could be moved out of the constructor but we need to set it as a class property anyways..